pkg/brokerapi/schemas.go

```diff
@@ -39,5 +43,5 @@ type ServiceBindingSchema struct {
 // InputParameters represents a schema for input parameters for creation or
 // update of an API resource.
 type InputParameters struct {
-	Parameters interface{} `json:"parameters,omitempty"`
+	Parameters *runtime.RawExtension `json:"parameters,omitempty"`
```

Comment:
We really cannot use the k8s types here.  This whole package is going to go away and be replaced with a dependency on https://github.com/pmorie/go-open-service-broker-client, so we need to rethink the approach to this fix.